src/plugins/maps_legacy/public/map/service_settings.test.js

```diff
@@ -281,6 +302,25 @@ describe('service_settings (FKA tile_map test)', function () {
         expect(tilemapServices).toEqual(expected);
       });
     });
+
+    describe('when unable to access OpenSearch maps service', function () {
+      const expectedDefaultTmService = DEFAULT_SERVICE[0];
+
+      async function assertObject(actual, expected) {
```

Comment:
Nit: This does not need to be an async function since you aren't awaiting anything inside it

Reply:
thanks, updated.